inc/ZipProvider.php

```diff
@@ -127,6 +127,31 @@ public function generate_zip_file() : bool {
 			return false;
 		}
 
+		// Build a mapping based on where the translation entries occur and separate the po entries.
```

Comment:
`ZipProvider` should be refactored to only handle building the ZIP files. A new class, `ExportProvider` for example, should handle the exports.

Related: #76